stdlib/3/stat.pyi

```diff
@@ -62,8 +62,8 @@ UF_IMMUTABLE = 0
 UF_APPEND = 0
 UF_OPAQUE = 0
 UF_NOUNLINK = 0
-#int UF_COMPRESSED # OS X 10.6+ only
-#int UF_HIDDEN     # OX X 10.5+ only
+UF_COMPRESSED = 0 # OS X 10.6+ only
+UF_HIDDEN = 0     # OX X 10.5+ only
```

Comment:
Is there a `sys.platform == "darwin"` check you could do to only expose this on OS X?